lib/cpp/src/thrift/transport/TFileTransport.h

```diff
@@ -352,7 +352,7 @@ class TFileTransport : public TFileReaderTransport, public TFileWriterTransport
 
   // To keep track of whether the buffer has been flushed
   Monitor flushed_;
-  volatile bool forceFlush_;
+  bool forceFlush_;
```

Comment:
Thanks for catching that - I pushed an update changing this to a boost::atomic<bool> type.